user-management/src/main/elm/sources/DataTypes.elm

```diff
@@ -63,6 +79,10 @@ type alias Model =
     , hashedPasswd : Bool
     , isValidInput : StateInput
     , authzToAddOnSave : List String
+    , providers : List String
+    , shouldAskPassword : Bool
```

Comment:
I don't think shouldAddPassword / shouldCheckboxBeEnabled and isChecked should be kept in the model 

They should be computed from providers list were needed (I think in view)

Reply:
Login / password should be stored in the PanelMode object